recipes/pdf_recipe/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "diffpy.pdfmorph" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/diffpy/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 35a9c5803b0bb3e4f4487544efe44d4063c38cc3601c33c893182359db4ece8d
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run_constrained:
+    - python 3.*
+
```

Comment:
This needs runtime requirements

Reply:
I thought that's what run_constrained was?

Reply:
https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

Reply:
I didn't see any mention of runtime requirements in that link - does it have to do with the entry points? Sorry for all the confusion about this